dbt/models/mart/all_available_listings.sql

```diff
@@ -0,0 +1,20 @@
+{{ config(materialized='table') }}
+
+WITH all_listings AS (
+    select * from {{ ref('stg_listings') }}
+),
+
+avail_cities AS (
+    select * from {{ ref('dim_city') }}
+),
+
+avail_listings AS (
+    SELECT 
+        all_listings.*
+    FROM all_listings
+    INNER JOIN 
+            avail_cities
+    USING (standard_city)
+)
+
+SELECT * from avail_listings
```

Comment:
Keeping it the DBT way allows for more changes over time too by putting logic into the CTE 👍

Reply:
Yes. I wasn't certain what the pros/cons for using CTE's were behind the scenes. It seems especially excessive when using just one table. But with more tables, it seems like it could be more readable by using contextually relevant aliases without cluttering up the query.